src/js/components/Anchor.js

```diff
@@ -52,20 +53,32 @@ export default class Anchor extends Component {
       children = this.props.label;
     }
 
-    return (
-      <this.props.tag id={this.props.id} className={classes}
-        href={this.props.href}
-        target={this.props.target}
-        onClick={this.props.onClick}
-        aria-label={this.props.a11yTitle}>
-        {icon}
-        {children}
-      </this.props.tag>
-    );
+    if (this.props.reverse) {
+      return (
+        <this.props.tag id={this.props.id} className={classes}
```

Comment:
There's a fair amount of code duplication here. How about something like:

const first = (this.props.reverse ? children : icon);
const second = (this.props.reverse ? icon : children);